Source/Core/CommandLineOptions.cs

```diff
@@ -1822,7 +1822,7 @@ checking errors
                    s = debug statistics
                 0..9 = number of iterations before applying a widen (default=0)
   /noinfer      turn off the default inference, and overrides the /infer
-                switch on its left
+                switch on its left (default)
```

Comment:
Also, above it says `The default is /infer:i`. That is not true any more, and it should be removed.
You can say something like `If this flag is omitted, Boogie does not attempt to infer invariants.`